### PR TITLE
patch-2.0

### PR DESCRIPTION
**Pull request by Agunderman10**

### CODE REVIEW

File: cv/index.html [hunk 1]
```diff
@@ -112,15 +112,13 @@
...
+                  <p>Hello! My name is Aryan Ebrahimpour. I am a Software Engineer and Designer.</p>
                   <p>
-                    I began programming in middle school and ever since then I have been interested in this field. I've
-                    started developing
-                    applications and writing articles in my weblog. I read software and hardware articles and follow
-                    Programming
-                    languages and technologies documentation to improve my programming and algorithm design skills. I
-                    always
-                    strive to become a better software engineer and I will never stop.
+                    I began programming in middle school and since then my interest has become an obsession.
```
Comment (avestura) [inline cv/index.html:117]: I like this part xD

File: cv/index.html [hunk 2]
```diff
@@ -598,7 +596,7 @@ <h5 class="mb-0">
                             <div class="card-inner-content">
                                 I can create Plug-ins for Microsoft Office products. I've already created an extension for
                           Microsoft Excell that can
-                          modify a sheet for easier question importing for Moodle. This project was a part of Sabnaa
+                          modify a sheet for easier question importing for Moodle. This project was a part of my Sabnaa
```
Comment (avestura) [inline cv/index.html]: The Sabnaa project was not a personal project tho, I think "the Sabnaa" is better.

File: cv/index.html [hunk 3]
```diff
@@ -851,9 +849,9 @@ <h5 class="mb-0">
                             <div class="card-inner-section">Summary</div>
                             <div class="card-inner-content">
                                 I'm experienced in Threading, Socket Programming and Desktop Programming with Java (JavaFX).
-                          Almost in all of our university projects, we were forced to use Java, therefore all my AI
+                          In almost all of my university projects I was forced to use Java, so ergo all of my AI
                           projects, PL design projects and Advanced Programming course projects are written in Java.
-                          I'm not a fan of it though!
+                          I'm not a fan of it!
```
Comment (avestura) [inline cv/index.html]: Why "though" is removed?

File: cv/index.html [hunk 4]
```diff
@@ -1136,8 +1134,8 @@ <h5 class="mb-0">
                             <div class="card-inner-content">
                                                           Visual Studio is a great tool for developing software. It has a lot of features that you can
                           use to refactor your code or debug your application in a faster way. I'm also familiar with
-                          Visual Studio Blend and
-                          it's great XAML designer.
+                          Visual Studio Blend and I believe that
```
Comment (avestura) [inline cv/index.html]: I'm thinking of "Visual Studio Blend and its great XAML designer".
It's --> Its

File: cv/index.html (hunk 1 above)
Comment (Agunderman10) [reply cv/index.html:117]: Me too haha. Happens to be true.

File: cv/index.html [hunk 5]
```diff
@@ -1535,9 +1533,9 @@ <h5 class="mb-0">
                 <div class="col-md-9" data-aos="fade-left" data-aos-offset="50" data-aos-duration="500">
                   <div class="card-body">
                     <div class="h5">Freelancer</div>
-                    <p> As I am a student in a university far from city center, I can't work in a company, but instead
+                    <p> Because I am a student in a university far from a city, I can't work in a company, but instead
                       I
-                      work as freelancer and do projects time to time. I build desktop and web applications right now.
+                      work as freelancer and do sid projects in my spare time. I build desktop and web applications right now.
```
Comment (avestura) [inline cv/index.html]: Is "sid" a typo here? Shouldn't it be "side"?

File: cv/index.html (hunk 1 above)
Comment (Agunderman10) [reply cv/index.html:117]: Are you going to merge the request or is there something else you would like me to add/change?

**Review by avestura — CHANGES_REQUESTED** [review]

File: cv/index.html (hunk 1 above)
Comment (avestura) [reply cv/index.html:117]: Sorry, I forgot to click on "Submit Review" yesterday

File: cv/index.html (hunk 5 above)
Comment (Agunderman10) [reply cv/index.html]: Yes, whoops I'll fix it haha

**Review by avestura — APPROVED** [review]

**Comment by avestura** [comment]
LGTM !

**Review by Agunderman10 — COMMENTED** [review]
Alright, I made all of the changes you requested.

**Comment by avestura** [comment]
Thanks again Andrew for your changes.

**Comment by Agunderman10** [comment]
Happy to help. Let me know if you need help with anything else!

**Comment by avestura** [comment]
Sure, your help is appreciated.

**Pull request merged**